robottelo/constants/__init__.py

```diff
@@ -631,16 +631,16 @@
         },
         'rhel8': {
             'id': 'rhel-8-for-x86_64-baseos-kickstart',
-            'name': 'Red Hat Enterprise Linux 8 for x86_64 - BaseOS Kickstart 8.4',
-            'version': '8.4',
+            'name': 'Red Hat Enterprise Linux 8 for x86_64 - BaseOS Kickstart 8.6',
+            'version': '8.6',
             'reposet': REPOSET['kickstart']['rhel8'],
             'product': PRDS['rhel8'],
             'distro': DISTRO_RHEL8,
         },
         'rhel8_aps': {
             'id': 'rhel-8-for-x86_64-appstream-kickstart',
-            'name': 'Red Hat Enterprise Linux 8 for x86_64 - AppStream Kickstart 8.5',
-            'version': '8.5',
+            'name': 'Red Hat Enterprise Linux 8 for x86_64 - AppStream Kickstart 8.6',
+            'version': '8.6',
```

Comment:
Should the changes in this file really be in this PR? It's an automation-wide change.

Reply:
updated the pr.